<?xml version="1.0" ?>
<!-- Blank world used for taking thumbnails of models -->
<sdf version="1.6">
  <world name="blank">
    <plugin
      filename="libignition-gazebo-user-commands-system.so"
      name="ignition::gazebo::systems::UserCommands">
    </plugin>
    <plugin
      filename="libignition-gazebo-scene-broadcaster-system.so"
      name="ignition::gazebo::systems::SceneBroadcaster">
    </plugin>
    <gui fullscreen="0">
      <!-- 3D scene -->
      <plugin filename="GzScene3D" name="3D View">
        <ignition-gui>
          <title>3D View</title>
          <property type="bool" key="showTitleBar">false</property>
          <property type="string" key="state">docked</property>
        </ignition-gui>

        <engine>ogre2</engine>
        <scene>scene</scene>
        <ambient_light>1.0 1.0 1.0 1.0</ambient_light>
        <background_color>1.0 1.0 1.0</background_color>
        <camera_pose>-6 0 6 0 0.5 0</camera_pose>
      </plugin>
      
    </gui>

    <light type="directional" name="sun">
      <cast_shadows>true</cast_shadows>
      <pose>0 0 10 0 0 0</pose>
      <diffuse>1.0 1.0 1.0 1</diffuse>
      <specular>0.2 0.2 0.2 1</specular>
      <attenuation>
        <range>1000</range>
        <constant>0.1</constant>
        <linear>0.001</linear>
        <quadratic>0.0001</quadratic>
      </attenuation>
      <direction>-0.5 0.1 -0.9</direction>
    </light>

    <light type="directional" name="front">
      <cast_shadows>true</cast_shadows>
      <pose>5 0 3 0 0 0</pose>
      <diffuse>1.0 1.0 1.0 1</diffuse>
      <specular>0.2 0.2 0.2 1</specular>
      <attenuation>
        <range>1000</range>
        <constant>0.1</constant>
        <linear>0.001</linear>
        <quadratic>0.0001</quadratic>
      </attenuation>
      <direction>-0.866 0 -0.5</direction>
    </light>
    <light type="directional" name="rear">
      <cast_shadows>true</cast_shadows>
      <pose>-5 0 3 0 0 0</pose>
      <diffuse>1.0 1.0 1.0 1</diffuse>
      <specular>0.2 0.2 0.2 1</specular>
      <attenuation>
        <range>1000</range>
        <constant>0.1</constant>
        <linear>0.001</linear>
        <quadratic>0.0001</quadratic>
      </attenuation>
      <direction>0.866 0 -0.5</direction>
    </light>
    <light type="directional" name="side">
      <cast_shadows>true</cast_shadows>
      <pose>0 -5 1 0 0 0</pose>
      <diffuse>1.0 1.0 1.0 1</diffuse>
      <specular>0.2 0.2 0.2 1</specular>
      <attenuation>
        <range>1000</range>
        <constant>0.1</constant>
        <linear>0.001</linear>
        <quadratic>0.0001</quadratic>
      </attenuation>
      <direction>0 0.9 -0.1</direction>
    </light>


  </world>
</sdf>
